packages/calypso-products/src/is-marketplace-product.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Internal dependencies
+ */
+import { YOAST_SEO } from './constants';
+import { formatProduct } from './format-product';
+
+export function isMarketplaceProduct( product : any ): boolean {
```

Comment:
I don't think it's necessary to accept `any` here. You know exactly what type of object you expect: `{ productSlug: string }`.

Reply:
TIL! Didn't know this will accept any object containing productSlug as a prop. 💯